botocore/parsers.py

```diff
@@ -507,18 +507,18 @@ def _handle_timestamp(self, shape, value):
         return self._timestamp_parser(value)
 
     def _do_error_parse(self, response, shape):
-        body = json.loads(response['body'].decode(self.DEFAULT_ENCODING))
-        error = {"Error": {}, "ResponseMetadata": {}}
+        body = self._parse_body_as_json(response['body'])
+        error = {"Error": {"Message": '', "Code": ''}, "ResponseMetadata": {}}
```

Comment:
Yeah I think so. I am just saying if there is an issue in the future in the way we parse for a message and code, it may get covered up by having defaults such as with this line:
```py
+        error['Error']['Message'] = body.get('message',
+                                             body.get('Message', ''))
```
But yet again, I am fine with defaults.